assets/data/effects/wildermyth-drauven-pcs_outpostCelebration.json

```diff
@@ -0,0 +1,187 @@
+{
+"id": "wildermyth-drauven-pcs_outpostCelebration",
+"info": {
+	"dataVersion": 1,
+	"sourceFile": "wildermyth-drauven-pcs_outpostCelebration",
+	"modId": "wildermyth-drauven-pcs",
+	"author": "Morgan"
+},
+"type": "TIDING_MISC",
+"verb": "MANEUVER",
+"ability": {
+	"icon": "heroism",
+	"priority": "1",
+	"cooldown": "oncePerGame",
+	"encounterEnabled": true
+},
+"targets": [
+	{ "template": "EVENT" },
+	{ "template": "INJECTED_HERO" },
+	{ "template": "COMPANY" },
+	{
+		"role": "hero2",
+		"template": "HERO_BY_SCORE",
+		"scoreFunction": "drauven",
+		"scoreThreshold": "1",
+		"notAlreadyMatchedAs": [ "hero" ]
+	},
+	{
+		"role": "hero3",
+		"template": "HERO_BY_SCORE",
+		"scoreFunction": "human+SNARK",
+		"scoreThreshold": "100",
```

Comment:
A few notes on this score function:
* Drauven and human heroes both get the `human` aspect - there's a bunch of stuff in the game that seems to be hard-coded to that `human` aspect to make the character playable, so unfortunately we have to keep it on them. 
* We are adding a `nonhuman` tag to all of our Drauven characters, so to explicitly require a human in a role we can use the `aspectValues` field to forbid `nonhuman`s from the role:
<img width="373" alt="image" src="https://user-images.githubusercontent.com/86335963/193594160-a41e7749-3708-4d86-bc28-51fdd06fc7c4.png">
* A basic aspect like `human` will only ever have the values of 0 or 1, and adding 1 to a stat that can go from 0-100 won't do much. This is why you'll often see basic aspects multiplied by a large number when used in a score function.
* If you move the are-you-human check to the aspect filters, than you can greatly reduce (or even remove) the snark threshold - even the snarkiest characters would have trouble reaching 100 in their snark stat. :)